### PR TITLE
[PW-4606] Add the threeds2 in the action deserializer

### LINKED ISSUE #431: [PW-4606] Missing payment response action - threeDS2
When testing 3DS integration on the latest version, there is a null target error on making a payment.

image

On investigation, in PaymentResponseActionConverter.cs there is no option for "threeDS2" in the switch which I believe causes the null target error. 

It doesn't appear to be compatible with API v67

To see the issue you just need to make a payment using the 3ds test card from your documentation. 

```
Adyen.Service.Checkout checkout = GetCheckout(this.currentEndpoint, apiKey);
//This is where it breaks
Adyen.Model.Checkout.PaymentResponse response = checkout.Payments(paymentsRequest);
```

Or try converting the below JSON to a PaymentResponse.

`var res = JsonConvert.DeserializeObject<PaymentResponse>(paymentDetailJsonString);`

This is JSON from the above request containing the "threeDS2" type

`{
   "resultCode":"IdentifyShopper",
   "action":{
      "paymentData":"Ab02b4c0!BQABAgBqShefxaKSYhF\/FJ99af+muEP7A0ZXQEoygJaA2mSwmAreKJwDxZ10\/Lhh\/lrGc+c9ilZ5KEk\/eZ\/+wD9S8svnS7HJmy9anMEVZR0NKaQMna2m4iY2AiFEbeC9oTCwvHBFBfmAHIk4WehqByKimDI8BBFLV\/A1XX\/f1m5jGTWVjeBmulSLIBqEL6n+8HinEwkRTDmjGU0lSe2sJo4eazAM5rpEn6WiOYh08CJX1t+QhhVmYWqz5sbaIVTykxRT3HUs6hB0SVsaWNaswvnks64OvzI00P8U\/9rmcXgpdtcxBka6DLmqhhAWwTuCslQH7TzXL40EqW8MgfH2tZ7MBjwev0zKXqLXyBpRuoSHfDNWhjbbWBacd5+q6SAd0IsZ0aSG\/OJQRGeJZbcJy\/qO3Zentc80X3eqiKMdfzRZBgNfa+87SSS9HElS5+u3CZTPRF0dqkF0xmvXKX9pRyGdaZ1aEwOVVGkwexqiqexD1p5E7X89tAeDKT2VQg+2OxC\/ovPt0Gr6N5taq3f6WNmwANCVcFeTmpAahXV2VYCDJiO3qJrW\/aQhAg\/qAW7RIYjW8QffzqYbJ7AgBeP5f9WNOEg3U2FCBuDLM9Fd1cL6KZswlMywrX8GenSoDcDRLS4vquF4N89g\/4KmUEvImPfb4upteITmef15Ygg451kOg8oyWBDPbD6FL0cPe10BpvF4KOToAEp7ImtleSI6IkFGMEFBQTEwM0NBNTM3RUFFRDg3QzI0REQ1MzkwOUI4MEE3OEE5MjNFMzgyM0Q2OERBQ0M5NEI5RkY4MzA1REMifV1TsVOXak8RoK51QaIQhXRnhuh+oWggjJISjMq55jLoy6OI7xLYbnUCgZsp7OcjWnLg3mXV+vddYxDF4M5IKK7FBBPC7pSvAD5e4mLncujHov5+SinViFnM9YXaE4Wzh\/+u9XbdulSqrZRedhGIT8OAnjJa1j8OIv1PUUDVOsa\/BD\/4OX84y0jzio2yyvzRCsUKH6yIEuq6ov95aDTH1MShkb4qegdPFJDfKUgBxX3Ev3RwDMp51YmnHYlTsaqOBpIE2mOQrOXswciZ3hfnAxUCbKLYBWZsARyJ1ZZYY\/H1l8b2s3dwjg0ykHe10dsBjzWsuvV4KdPvlDBHrWcf7FgLNPpAPjRJrrrOwppkRqGa7JxzbSlIywsDWSZEPgrpT230NSr7yVPAyyPyK9sJjKb9oE1354KTbG5nDpfrVKeruk1OfOWJCkD81HkKmdzHVMJyPth3StqTfpH9nFq2L4GoM7TFvCdL0L9RdbFE\/fgX8t88RFThNoDjX3FC8gj4PMur8fdbbRbZ0b\/oHpoK\/iSgzVvZxSHQi+XzFN+M\/XAmS3kTMnFyYgSIkG+4sHMPCPiqRMXlt64ycDy4B+TUYjLxUV7+SnVViUyIOrV\/4pWY\/\/rcvKplG+P\/pAUSIKZZI8rEFABbgQs\/xwZEGbR37hO9GCFurBFqiCzZvTHZrxGeO89hzTpXgWSEhbm4p+jeEI9XjNSvqbZVhPuixO5bR4E\/sO7K\/3PgXFzVZMW80qhCJFkx8jREQ0rIWa25tBZvTHnpExV3EkNxY3iUsS1HDWUna3bI0vNlx8gAsUJXMHAdT2aArFSfpMVkAgziNeVK\/icTMv4G+3jNLYpZCnnSnDDtsGZdKMPHRxAl9zzDU32r4DcAHldcWG4LXFuBkXdx\/M7BFqF95Dxl8H91kSKJb4mkRTU06MEOzaePkFLTQgams6D\/kHupB7DZxjy1s6+J3vRpdhzrMvBQVGroHsF8BkoOw8Qx29opgSKaqe+7EiOvG0f0lG3S4jiVVP916DzgkaaUNdj516UpN6qlQ7ZWw7VzpnV5wu5hJ6IQtNQvOJhegXeX2iTxoGHLloMJzRTZQ6XI390ZYqGng5jniDVPJkFsUqVDVYily7JlFZ7mSX420MWWwl0p+G9Yo9cFjCpn9lmeBQ3eFkuP\/Pe9TSgogN+tb1PgHiTIsyAU5OKoZAKmmABTTlMVLxCzjb0LjcZXzFyVfHabdJp65L9ichsKZti1DsxdQhqU+CihKocwsKqqh+PAUTDKnOX6hCOqGv14W66c4Phu0Tw5w+LRHeA4jTcqcMPnOI2vacfh2LhzOcPEuzog6X2QDCnZKQxoy1eD2VOvR9w+DPI45Xt4E2QjQf1ktbQNzmhKYlf4UTkCPJJf9tSiFHvW9Pf7MrT6lKKAYXUyEeIdlJ7Ipc7Gowyqmc+X1+JZ4iwNp+u6x0GaVl7wbuPSbrR8CCoz1+YNeS4DjOYajabhU\/GNjNgGelUN1l+TdHs2MEYV7iRyIfIos62Hj0AP63Nj2hGAs9I80\/4ff8fwrO91eWz4puMe+LiGcbPR79UxLUXcSmtH0DCBKPT4v\/6bAZATTwz4ZRF7+VJlZcBvqgsRmChbUR3Ep6jZXcG04giMMV1aiRyhKjsrHW0byUt4uVx14mWrn7QcoRueR4X+eGf9Oubc5mEh5ECVfL6YQULWNxyI7z+LhOW6zCWDo2J27BGQH8o0z6p+UXhLX4du03GhgN24V5NKDrHpjLvw83V9obQcQ2vXbkC\/nK+Jh0YrELDWRLsaQqi9ZTmI\/0ioGoMJXSi7Rp2FZU\/MxmBtaKQIKMp0DfvCd614LiEiGtfq93VvHJhfMcxdZpSHHDbaF58bBn5u6QhL3j5hdGl8ic2DceuwShOP7Xz1NksIdbMife6cjL3WeDzvbKSaPf3OW30ThZlTFzmjHPJ+ZZpTYK75Lj5DBiZYlMYLzR3WM9NkcrxFtYnIhMsWqbgxRIx3V\/Iz1k9RLD4QW7CuCb8ZD8oq4xjyqueNAljbVilFFZdMj+\/muS10afbIHNawYZAg1yw4\/X6IHUU5+48vf1znUSOJA1g9VazULjFVJHPREj6SxjIO+gP\/hA4\/QR4x3+IMymC4nYKaE+OBnOaSCT\/ltKTUYX+MK30eUPUt4Wky0m1c8Fed4ALS4OItM+iCJcJ+wGeWcbxQvMOJKD\/qWr+IAD8M7JvtSio0IjJWkSeeR\/ob4GiCoCx+4Ro0KrnHz4BizHNq7T3lwNozipYVamQPl1rrIAzFDnScgnkIEz1kadv5rcZFDlxqBi8\/5S5oKdO1St3EEEur9VVgv4isNpOLZz4vJ8khp46QtA1mvhraJKS6LGszMvbEaOnUgbyFvB8ZZdTdOMd4pQn901\/p3dzLbxUONnRe++tZqRMsq6xWgMWyM+cxM21l1iGsVoGiGBOhBXlHFpaKpx61XuSmQAgAR+kiC6stBdyHjZKi3MQlXFV9\/lD6KDrByLLi9u4o1UB63YzCsADbgRbFH4jdxizH0IolPCPFtNBfjX1KVRgmqrotihpAdWBQkg6I3h7ZL7gcTo8jbPSF9cihNaySkIYzQucFH2uNHHtnTCuiVNRzQYxIg5Sif\/sObueJ2xUuuoKk1LSDlwmdZ5IKZxyjwScel2Te\/Ls3AMtrQqDeOWD+CUocTAobEklZ9HixRKnEvmbLq6z13DbigzA\/pn4EPBGdsQHfIr4JRXCYR6YkRNhxMXJyQqPYFgMfJEnZl1lL\/\/DiNCB4A2HureQLCENgPsyKnS+JmEt8stn1s\/hAUelrEaC+UvKMVT12rzuQT2uCmzGuvE53r4iJOMo5p3ROT8dbDH5K2uC7UfeQHpKD\/FxD5gADwg3FCrDYYm02QTxZKkVwGuB4dGF4Z4\/ZcB\/4ZTtIWCngFVt+Tp1u+qZoeo62\/KZIBs7\/PmKZSnfU0ba3KW8n0+00MPlXnFIUU1ZudsgheKH\/L3x5EtyjXimyHsylwJBKPwpiUyoqZeRz\/wB3KZMQzf8ZV\/CwCnMvHn8K6FLnXvyyMNbp1iG0gRJ3OyN3ilW9Bd9q9lAXM5l7+ZNsSYCybt4dqOsBez7MO4LdVsFMpK+VYzW\/gd+5XCbpKNL+q5istGi\/BpEhcsANBecgNFqCNHsvfAsv+F4yV9VDd++uHICJn+Vhri4ef2ZW\/YOp9GysgobMm7plujmiQnD565uxL5JcPc++5l8hR7NPzef5BO1Zf8rc1S9AGSlntWX3VApc\/WKHyHOyX2NU37C3n1ESHwEFVDYHLf8DlfWp7bJUwyFPVVMZji0lHYrkIJRgblkWs49sxT99AcFDwD\/tllKH8KodnRy4X+8W\/uwGldyKtgE6jQ7OPEvPA8PsllimIBP1z+wXlhjMyJARFZwIfI6P6ZuuEDVoRiK8\/jHKS29ta\/O99UTmQAIsfNKU4WML0boKkBHIRk3foC3zb+RFMQwIHRvqw5EUrvgb39c1Q1e9hX5sZnYX38ZooQlcMo+h4wfMi9tTsDxxgJGEVI4KBsSGm7jkjOObwRV3WzbaLuheARXoyAFTx\/Go24nnqRRS9bCw2o1vydsBgeId3mb0C9j8ld1cllos6PuzM8khM3Th7kmze1NDkHbswZzrJSinuzDqVDCwcOTEZasGeMF9DveBecaVa4+Qd+u+j4BB8iwQyJAI+oYFlZ1ghVqnAIo3ihxCWrLAZuzfz2gV4f91BKKEih1wAubXsYQGJ5XvreMif7lPs4yNiIbQSSoYh4pert9YqTVVzmu6BLOkBL4t2aasuhYVJNVPqKlnkOnFVpj48ucOEjfNSTl5vnnRUQnd9EXkdR3wNkGjIq5iFcophNTASvrguBTqE4Sc+mvbs83Yeyb2yWu53IdLPSRUIV3iY7BJzBPx29Gr\/zT6jQ6xkhFcl0wC73Z3brtzmK+0KpAX+1wtz7rrjBVnuqlfxAmHKFMeXiXzpU1RIfdJdXG0gH\/YC2Ck7sKkpqoNl9s2bGxhq15FXouOn1PO3ul9BZTZGEZVEEsUhkhzmrHZaPrt29n1yYdvm2jmVKip+QVPyz5H3WhHxkzts4i\/Rn6ts12rv8Z+9J2Kz2TQdHuq80+LeOQ5pSv5dkEiOyR8lGIRFZJcemVOdMdHdaGJkMOIiiEEVjAyKvNjNNjCCAb+z8oV185ug9XRRNnTKlO6Ha9r46UsI8bq6AQBX2ZQ6n7qQxcXLXwzT5xINfDLPwH5WfM68rQzra\/qLXc6KkyzcygcYB0iKs0ykg\/Dzpap93WcJ4GVNTsyySo34LD7OXDKuiIJWPgbZ3T66s1Fvgju3SYUrB1zD6W9caDfySF2aruHF63xsi78JiS7FdVN1V7Ba+sKM0sqwnVocBMbZLZSH6\/IVuMDqaJWT5xI6xu6hMwWngJvpjHvLXmuSMn9\/CQD1Ly0+wGUGlFcWd40eCsjqMIq5UD4ks8wfTPj\/Fyz\/mWfn1pgbXWWuCzqeeT5T+ukvtOFGSPzB5zK8YichvOOHiEgg7eLgITgl8cxVBjNjZPUwnfGe0R6M\/pChrj9S4c91KaFRDy3YcHC\/M0LJY594bfF4u3+ru7FjUpTfyUBgbToFajMD4h8V0gC35TYjDGgmZqECP1VoYrkRHhByf6ARKhdvHVDWs6fYbCijL4iZb1kSOnIMF+3CTf45DdA+sjvIXveIF7w3do4gq8nV\/S6v0R9iBxieG6jYALyHSEqbu8zTC2ciMcqSqoJYntHg\/gou\/bj9SPDl4VKIZZohv\/JyRBfET+fHunV1gZnB5LHD6BlenWCSgDE+jehBDxOuHgekcRDVjZ5tKtklaBkE8v\/nKTljvGF71QyxmSm5ak0DwPwCIMeYVUpa5PL5TsmaXcekMpgkzgD4ZVyvFLOQJi6C0ndhcY8VHUP33G5f2s5BCVAMjvpdCX25hHa536WkRnTI2OkSPezK2fXXVoLdIL1S9p6pWOhDNOWIJ0Ie9lSaKFbMX1plLDmM9D8OWvFsYEXDjAdgiMJl\/vfOS6cm+z5qShddfghK6BVOVlJGdUlJ2D6c\/XfCcKFifv7bTYlWc\/6DttmHtxL7cD67El9O7iE58g\/SuLRJNJMltl4ZlR\/\/Uxha3\/80lfOFa3v0Xv6hz3OtkyiQ7Fa7egz85KyMVZiwqjKwHXkQmWa4SaVBMnlPWc6Rlo5t0umIS4t+vRYqxydprr7ySO7SUndgmWlPobRZkBGcLLkzkcvcEtMdMJA6RTataarULqhYfX5FE5kodULDnQ21MLSrH9F\/jNsCo3hK6mmfJBRQO+2gQA78VPVPSbh8XBGr60WDoThgFTJM5N+uY2pH\/BPxOB6bv0uIml4oBnqzwdzi7yAzNDFHNeZTaORHAL+1jEgkE\/WGDeQ8DIE111u9gtDVbhss7AVzG3DvQrBSXYTZwQbUsXFG7ZD6sQ0oT7vQURJDHp+CuU2a11CtDsXzJILTDPemnCmDVv\/CAfWYdVutlpwUiK6DoPjM7RLnmf6+NrwCiBajs00JGuSk5pCsncZJhMpug1FMV89BI2F56R1Naj8yvdHM8GcwOI04PIOHJ60v7FpuyNCzQWrOuNSg+GTw5F7Ys7n+qLeUdxueZY6ukRQVRzJXy9hr2tHAPUyxBZ5o6s4NgzlybR6mLxO13SujSEkv9RiJSC38dSmiEANZ32o7XOtxEDS6dXLRFQsCeQRsQXDjA1qiO2M0evP\/WeaUZWgvHIE0sbyOCI8hbxTdYaQE9Bbf\/o6b3JJz1gQPdhyTqKX7gy9YhD99MAvI6voujnzsu3+V1KWEf6gf8VRfOlCza1KTnRqgQ6KeGmvpdtV\/HMRFJqPZ8RE9eoA7ii9EKbqzazWd+nIkY8oCgFU1Jz3n1oz0S+HK+9+g+RUHDQ6PkIB8m+KCPQ0vz5Xvm+mIxt8x0DC0+YXXuPXc9fCRGfPF29sEFovHh3jMu8eZbfN0XOOgrIp7SxsZevYoV51rzch8syAebvgbFIDFyielFdzRw2VJlauJtyeOpl41yCAXNNJTTdfqyiAcSN6y7AgJ+7jwRda6FxjVhaQIvC5\/id0KPVh\/HBHhrk1t4mCVT6MvNymbfFSrAWLtAI0kNQJCu5nFe\/Joww\/o3rt7OiNv45Z8hVYfDIEg2IVvw8UD1izE4\/AuB+5p3QoIZ+9SrtkurJv6Z79w9MnMEouXRF+bD4l4Ey7nbxVf+Hqd\/w4QJLXmRx\/vJ6GWLQ5oBCwruTolONlRyHRlDfUdDzVG6l9od\/mDo9lVB1MUxRADDKLSaDcDkUy5MUuTmxzn4BlMBN9Ce+F1uVVgHv5YO47aUZtmynRXCze++38oxp\/1dz\/pMerVW5i6ElmX\/iFDXlXWvjCxgHTcxiv\/LB8l04FGoHHQqQQW5HuoSuqr41oHrkdBmU1pZUB7st+ZKPYNgyH9AdYo3hRVAvd\/vaZr1gdmwZDrzoWCwz9BICjtgdHksLjTZIImr+aO\/tXKb2Jq330Ri0RUEUREAbRUsUDiLMN8lEDzjQ4GW9y4f5Kxi1ahYI83M62YMBh9DiSlV2nr1\/fxx1VGJLzSB37fKXs4v40Nfcxnn6Bv+Mk7WwTP36+YP2Om16XUkDnwl0\/1nezKGhbXMP9wNeSWllplu4PHn0IdMCx8TG+6+iLl5Ym8G3s9ub4ShYYf\/TRdMVE1DBsndJeVwp8ixxDdAyXWvaYZZzjXgnz+vJFJ7GFTK4wku9uQEn8XSjIpklEMacQDxedgryHRxrbkszil6SaAGVYqhzw6RqEPnJbKLK+rHAh3SwkhCz6xtSkt1kYHx65U9huA7Q6fyG5SrSDpNhlhlQSa2PnKWG5z2e7gTAVm1bIbU63NvlkFWAmkQQzZxf1dzWfBj2\/E8YrLludgxMu27svgcogbvMrkKKQrQeHF8t0kYb\/yA3DFxE=",
      "paymentMethodType":"scheme",
      "authorisationToken":"BQABAQAtoCgu4f1F43TousQB6ayVPRw+IcQG535CB2TBnNvMbcYvrfqlog2ZyEomJqjNyn+3R\/hTLmCWVS3C09cok4bVbXdldMgWsXa1nT4GryjlL82chhVJyOSTOxOtCWvot25VMYmKQ2VvgzwB4S5p3UM4jK4HyAloRSdr+t\/Y25E72kcR2Hx3gY6uWkBivNrEfpfcWRAX1rBGX4Unpbp8QfCIunLeVouT76jYV\/XblktC59Jn8X4lZjsJ2\/y74D+vsdAvHHlPzjSJslkjD2oH1N2ej+S1Y\/ka62USC+aAr\/0spFsBnxoRQxZEgRxRoyy5+eriwAhXpsBRvAHr2T1UE\/IbEJmbx151OCqE7iNrMzosRXsAABEKVQBDzMHIjn3IIzeIvW2yHaOR3k1sFm4e2jUAOHTA5aoWqI+cAbk5xvrWWB09OiAjSClDVF0VThMBSD+ux7G30HJtrJ9jSZYFgbV85I1MPrY7le+k2vyqbRo5o96syIuJPMeYaYdhoM6zUt2UYUOPE4fbLAB0Tt4i2jf0NFx4pZQ\/WC8RiesQnwhJCWoOFcj\/PmU7Xim6xyB5cFrzRIeutYJS\/A8ChobdVpoqtt16R5voELy7KDH0V1R+PbQSAplgogBlvmC+q+tJUIuJ8UKdqFgi2oTzk7FM2xk\/a964DvC9+6vTDuET2TJyX67mLQaB\/+ProyK\/rRC\/7DoK9s4auUqkaw6C\/nfKyhk3buXg+hMBppPZ0L9w8Jfl4Ei+pj1Z0ZZhN7ZrKFKfXPW4Vv3BUmGnZuughjKdnSaFjIGzYKWHBWji97xNptxBfVDLOACD4h+hu1KAcZriirrXN94dYC9Zf8S1L5ORRWLAv566Y\/Qvfban2EeuF4nvz7HYAbXr+EBb1KRTAWHUSkkvQ4tFwCewUTyV2KRtoNm\/fkDnUAR2CfT7cKDwJ+H9P9aBgfE4zDQGcWtWotQc3BjgU9+ARGMBwHQ+Acbdo\/8pvBkgQyRDmDDn8If\/iTZxNtzkdI\/KRYeYqXDWN6YkG0HULi79HNvUPYuDGt6x1NEKaN2AiVA0iYGvR2dIfUO+4oUakeDmJSsr6FnMac12ywIOHET5SIjWQkwWh4yGD8eX9lVLG3p3BB2PMMYKiIvPNHjoNGIAuk0tPL41prtYSiKiSUjk8fQdRd\/XTBAMNySJfA5vQ4t9s1qMNPu8WsfuEZhuinMrFEvTRZh7oipyS6yRs8rCktmDpb8EuiY5yw5K5pZma7eqZe7kzO3gfhVV4lwqtBCIrtZjqrbB9UDw9+eUrseeOSs3Gqte5Ev0SgJVXJUsPI7BQ2wk14bT7ROKfDwRs0ylaR+jASOES7YKteHsm7dv3\/2Vk3iRYCbrrQmwkg8J3x3BdA0g3rRDWHaBoUsSOwdz2GVj\/3OUQlYSOQacszpTbxeEq9kEgd6qeTaV87zocRhAO5d62Cgs63TlPtWL9RmJoJgwexkh\/mXUtu9QzLCxHnT31KG1P2ibr07il8xbhW9yZTxHL8tDEb5KMPZ\/WjB\/hGv92ovCgVFT0Mu8vAH+NFIAMZ5rHGeguECPIa7xJd7pYHbQIfM2NwUefnkagnUQKTt5pZhcj34dlCHRhn7XyqQCFuesMSUg4ZoHOJR5t0TS32YU3UteDaydpYrlIUQ5nxE60yoxpc3FZEq3dK7nspCw4Ai6tgQEtcV3Qx4ySmLFitHQc9dhaSJMuG5aqumbDScaoLWerl4VfaatKH9zZU\/U0VUiis0nYrO6geRvV61W0ihSYci5VhQCn1ZPCcBknzrnxKogq7tkNQH+z9VobcCVWb3QIxa0K642CrPM5qPVKXIQXNsSwKTukrTOcksf8MxFhOziYITM\/UWtP\/kqU57pnM73fwD9Z0O\/oohOpfIpAXpm2U\/Z0k3A8ci2EKg\/9TD22+bPkx9Z2ZItf1OKBr4yWETO8SYWT+suacf6etVKp8xl28twoEDm2lyyZ2Z2dfic82vu4e5rXpmIs8M4Q6TX6nN9wJK14qhQoTD3ltvTu5SUY7GVWjdy41CZ0V6hzKvXigQ5swYeXJtWX3oR5i0CthjjG3688ut6u0z9jZ7eOktwXiu6CQiqqgvOURcCazHReJXQ0006sr19Rj8va5LDeespixLIfN3EQ+IhL6fyjlc9I3E4N7S0PN50+p12et5kcndybA067Hq1hkLP4huqfQCbSJDtvurjLwm\/FFGnCurscp4EjwBHcYEpf9Ju8QBWEwyQLkQEdAAMqQRqX6FFdokomj+OxDOp8o8JAszCeyeAiOMAikiouYKT3WmFSdFe7exKnog6frEP38Wetz66iFugL002fC3on2stHou1lbtiqQod3PJCDT\/1TYMa339\/Hfmkm3leKUbXCIY4jomp4KtA7ZoY7ppaKLQi6ZZb15g7dnVivsypv09+nG9MlkG7v1FbuY+cCXurjhZWs7BalwNzSTaQuz\/pRtg3HzGF9ieYhPfB7HUiWPN70sV99fjFx13nGZqMpuK6I88D6ffpGIyT9Frp4h2r5qy1XBj+\/RUyjcY8vhSKfetTm1HPZUKu1k2aTfm9R7BudHYUMCrBMKAXDMNODloZLkAvUY\/lVr5JLsMhAjXdOtv2JGRdKajWn0s8U8fszuzzNo9veKSF7f\/peQ295BnmTYoYj5gy1NwLNGcsON3nYlnh04kmGEAmXkcHKCCPpL4TtPYyH689kqA8un1awsGg9D6Fas92+meqM8vtAco=",
      "subtype":"fingerprint",
      "token":"eyJ0aHJlZURTTWVzc2FnZVZlcnNpb24iOiIyLjEuMCIsInRocmVlRFNNZXRob2ROb3RpZmljYXRpb25VUkwiOiJodHRwczpcL1wvY2hlY2tvdXRzaG9wcGVyLXRlc3QuYWR5ZW4uY29tXC9jaGVja291dHNob3BwZXJcL3RocmVlRFNNZXRob2ROb3RpZmljYXRpb24uc2h0bWw\/b3JpZ2luS2V5PXB1Yi52Mi44MTE1NjIwNTc5NDc0OTQyLmFIUjBjSE02THk5c2IyTmhiQzEzZDNjdWQyRjBZMmhtYVc1a1pYSXVabkkuX1FIZGFQUmEzYmRVR2V3ZmZDdjlJOFNRaDAwbV9oeldzQmlnejJwQTNTRSIsInRocmVlRFNNZXRob2RVcmwiOiJodHRwczpcL1wvcGFsLXRlc3QuYWR5ZW4uY29tXC90aHJlZWRzMnNpbXVsYXRvclwvYWNzXC9zdGFydE1ldGhvZC5zaHRtbCIsInRocmVlRFNTZXJ2ZXJUcmFuc0lEIjoiYjlhMWFkYTQtOTJmZi00MmZlLTkzYWQtNThiMzQwOTcyNzUzIn0=",
      "type":"threeDS2"
   }
}`

 - OS: Windows
 - Browser: Chrome
 - Version: 7.0.0.0

### PR DESCRIPTION
**Pull request by AlexandrosMor**
## Summary

In v67 for the native 3D Secure 2 authentication, the response now contains action.type: threeDS2. [Based on the documentation](https://docs.adyen.com/online-payments/release-notes#mar-1st-2021), this replaces the threeDS2Fingerprint and threeDS2Challenge parameters used in earlier version so the `Adyen/Util/PaymentResponseActionConverter.cs` should be modified. 

- Added: `threeDS2`, `bankTransfer` and `wechatpaySDK`
- Removed:  `threeDS2Fingerprint` and `threeDS2Challenge`

## Tested scenarios

Tested with unit test the `PaymentResponseActionConverter.cs`
**Fixed issue**:  
fixes #431

### CODE REVIEW

**Review by deepu105 — APPROVED** [review]
Lgtm

File: Adyen/Model/Checkout/Action/CheckoutBankTransferAction.cs [hunk 1]
```diff
@@ -29,13 +29,13 @@
...
 {
     /// <summary>
     /// CheckoutBankTransferAction
     /// </summary>
     [DataContract]
-    public partial class CheckoutBankTransferAction : IEquatable<CheckoutBankTransferAction>, IValidatableObject
+    public partial class CheckoutBankTransferAction : IEquatable<CheckoutBankTransferAction>, IValidatableObject, IPaymentResponseAction
     {
         /// <summary>
         /// Initializes a new instance of the <see cref="CheckoutBankTransferAction" /> class.
```
Comment (Aleffio) [inline Adyen/Model/Checkout/Action/CheckoutBankTransferAction.cs:41]: https://github.com/Adyen/adyen-openapi/blob/master/json/CheckoutService-v67.json#L3582

I think we can remove `paymentData` from this class

**Review by msilvagarcia — APPROVED** [review]

File: Adyen/Model/Checkout/Action/CheckoutBankTransferAction.cs (hunk 1 above)
Comment (AlexandrosMor) [reply Adyen/Model/Checkout/Action/CheckoutBankTransferAction.cs:41]: Indeed [commit](https://github.com/Adyen/adyen-dotnet-api-library/pull/434/commits/2a97c2cebb1135b099ca6afd3ca665381fdd2651)

**Review by msilvagarcia — APPROVED** [review]

**Comment by AlexandrosMor** [comment]
@deepu105 Lgtm `No new or fixed alerts` so there is not something to fix :)

**Comment by deepu105** [comment]
@AlexandrosMor I meant it looks good to me :)

**Comment by AlexandrosMor** [comment]
@deepu105 👍🏻 😁

**Review by Aleffio — APPROVED** [review]

**Pull request merged**